test/files/pos/t8111.scala

```diff
@@ -7,10 +7,10 @@ trait T {
     foo((u: Unit) => ma)
     foo(0, (u: Any) => ma) apply ()
 
-    // crash due to side effects on the onwer of the symbol in the 
+    // crash due to side effects on the owner of the symbol in the
     // qualifier or arguments of the application during an abandoned
-    // names/defaults transform. The code type checkes because of
-    // autp-tupling which promotes and empty parmater list to `(): Unit`
+    // names/defaults transform. The code type checks because of
+    // auto-tupling which promotes an empty parameter list to `(): Unit`
```

Comment:
Actually, autotupling is the wrong term here, since inserting the unit value is deprecated and autotupling is not (yet). I tried to start calling the unit value `()` a `nuple` but that never caught on FSR. But maybe `autonupling` is another opportunity.


>Adaptation of argument list by ~~inserting~~  ~~()~~ autonupling has been deprecated